src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -111,6 +112,41 @@ public static void assertCommandFailure(Command command, Model actualModel, Stri
         assertEquals(expectedAddressBook, actualModel.getAddressBook());
         assertEquals(expectedFilteredList, actualModel.getFilteredClientList());
     }
+
+    /**
+     * Checks if the {@code InsurancePlansManager} object works as intended when a command is called. This assert method
+     * is required as the {@code InsurancePlansManager} is mutable and has no equals method in {@code Client} to check
+     * for changes using the {@link #assertCommandSuccess(Command, Model, CommandResult, Model)} method.
+     *
+     * @param command                      command to be checked, must be only for Insurance or Claims related objects.
+     * @param actualModel                  Model that the command is to be checked on.
+     * @param expectedMessage              the expected success message.
+     * @param actualInsurancePlansManager  the actual state of the insurance plan manager.
+     * @param expectedInsurancePlanManager the expected state of the insurance plan manager.
+     */
+    public static void assertInsuranceCommandSuccess(Command command, Model actualModel, String expectedMessage,
+                                                     InsurancePlansManager actualInsurancePlansManager,
+                                                     InsurancePlansManager expectedInsurancePlanManager) {
```

Comment:
`expectedInsurancePlansManager` instead of `expectedInsurancePlanManager`, remember to edit the javadoc too 😃

Reply:
noted, fixed!